tests/unit/api/multipartUpload.js

```diff
@@ -89,7 +89,7 @@ describe('Multipart Upload API', () => {
             // until here
             const testUploadId = json.InitiateMultipartUploadResult.UploadId[0];
             const md5Hash = crypto.createHash('md5');
-            const bufferBody = new Buffer(postBody, 'binary');
+            const bufferBody = Buffer.from(postBody, 'binary');
```

Comment:
encoding is not a valid argument here as the input itself is a buffer. Here's the example from node's docs
```js
const buf1 = Buffer.from('buffer');
const buf2 = Buffer.from(buf1);
```